homeassistant/components/fronius/icons.json

```
@@ -0,0 +1,54 @@
+{
+  "entity": {
+    "sensor": {
+      "current_dc": {
+        "default": "mdi:current-dc"
+      },
+      "current_dc_2": {
+        "default": "mdi:current-dc"
+      },
+      "voltage_dc": {
+        "default": "mdi:current-dc"
+      },
+      "voltage_dc_2": {
+        "default": "mdi:current-dc"
+      },
+      "co2_factor": {
+        "default": "mdi:molecule-co2"
+      },
+      "cash_factor": {
+        "default": "mdi:cash-plus"
+      },
+      "delivery_factor": {
+        "default": "mdi:cash-minus"
+      },
+      "energy_reactive_ac_consumed": {
+        "default": "mdi:lightning-bolt-outline"
+      },
+      "energy_reactive_ac_produced": {
+        "default": "mdi:lightning-bolt-outline"
+      },
+      "power_apparent_phase_1": {
+        "default": "mdi:flash-outline"
+      },
```

```suggestion
      "power_apparent": {
        "default": "mdi:flash-outline"
      },
      "power_apparent_phase_1": {
        "default": "mdi:flash-outline"
      },
      "power_apparent_phase_2": {
        "default": "mdi:flash-outline"
      },
      "power_apparent_phase_3": {
        "default": "mdi:flash-outline"
      },
      "power_reactive": {
        "default": "mdi:flash-outline"
      },
      "power_reactive_phase_1": {
        "default": "mdi:flash-outline"
      },
      "power_reactive_phase_2": {
        "default": "mdi:flash-outline"
      },
      "power_reactive_phase_3": {
        "default": "mdi:flash-outline"
      },
```
or, if you prefer the device classes default icon for all, remove all
```suggestion
```